test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2076,19 +2087,25 @@ static public void testDefaultValueInLineByteCodeMultiArray() throws Throwable {
 	}
 
 	/*
-	 * Create Array Objects with triangle class without initialization
-	 * Check the fields of each element in arrays. No field should be NULL.
-	 * TODO:Fix with issue https://github.com/eclipse-openj9/openj9/issues/22642
+	 * Create a null-restricted atomic array of triangle2D.
+	 * The array must be initialized with an explicit initial value
+	 * and every element must contain that non-null initial value.
 	 */
-	@Test(priority=4, invocationCount=2, enabled=false)
-	static public void testDefaultValueInTriangleArray() throws Throwable {
-		Object triangleArray = Array.newInstance(triangle2DClass, genericArraySize);
+	@Test(priority=4, invocationCount=2)
+	static public void testInitialValueInNullRestrictedTriangleArray() throws Throwable {
+		Object initialV1 = makePoint2D.invoke(0, 0);
+		Object initialV2 = makePoint2D.invoke(0, 0);
+		Object initialLine1 = makeFlattenedLine2D.invoke(initialV1, initialV2);
```

Comment:
Same as the previous comment, please use different values for each point and different points for each line.